### PR TITLE
Update JWT changes in the U10 release-note

### PR DESCRIPTION
**Pull request by ayeshLK**

### CODE REVIEW

**Review by NipunaRanasinghe — APPROVED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -113,9 +113,10 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 #### `jwt` package
 
-- Add support to directly provide `crypto:PrivateKey` and `crypto:PublicKey` in JWT signature configurations. With this update, the `config` field in `jwt:IssuerSignatureConfig` now supports `crypto:PrivateKey`, and the `certFile` field in `jwt:ValidatorSignatureConfig` now supports `crypto:PublicKey`. These additions will breaks the previous union-type support.
+- Added support to directly provide `crypto:PrivateKey` and `crypto:PublicKey` values in JWT signature configurations. With this update, the `config` field of `jwt:IssuerSignatureConfig` now allows `crypto:PrivateKey`, and the `certFile` field of `jwt:ValidatorSignatureConfig` now allows `crypto:PublicKey`. With these additions, union types (or whatever that is not supported now) will be disallowed.
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:116]: or whatever that is not supported now -> I meant to rephrase this to specify what is not allowed now :) Not include it in the release note.

**Review by MaryamZi — CHANGES_REQUESTED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md (hunk 1 above)
Comment (ayeshLK) [reply downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md]: As we have already mentioned the previous and new types in the code-samples, IMO we can mention that previously specified union types are not allowed in the text. Updated the PR with following commit [1] 

[1] - [406b2bc3d4bcd3961a7856a1849d5eeab3441d61](https://github.com/ballerina-platform/ballerina-dev-website/pull/9221/commits/406b2bc3d4bcd3961a7856a1849d5eeab3441d61)

**Review by NipunaRanasinghe — COMMENTED** [review]
LGTM

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -141,15 +134,25 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
...
     public type ValidatorSignatureConfig record {|
         // ... other fields
         string|crypto:PublicKey certFile?;
     |};
 
-    // new `jwt:ValidatorSignatureConfig` record
+    // New `jwt:ValidatorSignatureConfig` record.
     public type ValidatorSignatureConfig record {|
         // ... other fields
         string|crypto:PublicKey certFile?;
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:146]: Isn't this the same as L140?

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 3]
```diff
@@ -141,15 +134,25 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
...
     |};
     ```
+
+>**Note:** This feature will break any code references that use the previous union type.
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:150]:
```suggestion
    >**Note:** This feature may break existing code if the relevant fields are referred to using the previous types.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 4]
```diff
@@ -103,19 +103,12 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
...
-
-### Ballerina library changes
-
 #### `jwt` package
 
-- Add support to directly provide `crypto:PrivateKey` and `crypto:PublicKey` in JWT signature configurations. With this update, the `config` field in `jwt:IssuerSignatureConfig` now supports `crypto:PrivateKey`, and the `certFile` field in `jwt:ValidatorSignatureConfig` now supports `crypto:PublicKey`. These additions will breaks the previous union-type support.
+- Added support to directly provide `crypto:PrivateKey` and `crypto:PublicKey` values in JWT signature configurations. With this update, the `config` field of `jwt:IssuerSignatureConfig` now allows `crypto:PrivateKey`, and the `certFile` field of `jwt:ValidatorSignatureConfig` now allows `crypto:PublicKey`.
+
     ```ballerina
-    // previous `jwt:IssuerSignatureConfig` record
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:118]: I would break this into separate code blocks and move the comment out.

````
Previous `jwt:IssuerSignatureConfig` record:

```ballerina
public type IssuerSignatureConfig record {|
...
```

New `jwt:IssuerSignatureConfig` record:

```ballerina
public type IssuerSignatureConfig record {|
...
```
````

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md (hunk 2 above)
Comment (ayeshLK) [reply downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md]: Fixed with [d36e88eb689b633b5ad1a83b5218d56333650449](https://github.com/ballerina-platform/ballerina-dev-website/pull/9221/commits/d36e88eb689b633b5ad1a83b5218d56333650449)

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md (hunk 4 above)
Comment (ayeshLK) [reply downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md]: Fixed with [aacbfc27781dbea4f9c8dd473e17201240579cc6](https://github.com/ballerina-platform/ballerina-dev-website/pull/9221/commits/aacbfc27781dbea4f9c8dd473e17201240579cc6)

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 5]
```diff
@@ -103,53 +103,62 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
...
+        |};
+        ```
+
+    - New `jwt:ValidatorSignatureConfig` record.
+        ```ballerina
+        public type ValidatorSignatureConfig record {|
+            // ... other fields
+            string|crypto:PublicKey certFile?;
+        |};
+        ```
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:154]:
````suggestion
    Previous `jwt:IssuerSignatureConfig` record:

    ```ballerina
    public type IssuerSignatureConfig record {|
        // ... other fields
        record {|
            crypto:KeyStore keyStore;
            string keyAlias;
            string keyPassword;
        |} | record {|
            string keyFile;
            string keyPassword?;
        |}|string config?;
    |};
    ```

    New `jwt:IssuerSignatureConfig` record:

    ```ballerina
    public type IssuerSignatureConfig record {|
        // ... other fields
        record {|
            crypto:KeyStore keyStore;
            string keyAlias;
            string keyPassword;
        |} | record {|
            string keyFile;
            string keyPassword?;
        |}|crypto:PrivateKey|string config?;
    |};
    ```

    Previous `jwt:ValidatorSignatureConfig` record:

    ```ballerina
    public type ValidatorSignatureConfig record {|
        // ... other fields
        string certFile?;
    |};
    ```

    New `jwt:ValidatorSignatureConfig` record:

    ```ballerina
    public type ValidatorSignatureConfig record {|
        // ... other fields
        string|crypto:PublicKey certFile?;
    |};
    ```
````

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 6]
```diff
@@ -109,35 +109,13 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 ### New features
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:110]: Sorry, one last change. This should probably come under improvements rather than new features?

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 7]
```diff
@@ -109,35 +109,13 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:109]: Should go under library updates, not package.

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md (hunk 6 above)
Comment (ayeshLK) [reply downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md]: Fixed with [21ba8dd8c3a061eb1718a6a5894de146dace15a0](https://github.com/ballerina-platform/ballerina-dev-website/pull/9221/commits/21ba8dd8c3a061eb1718a6a5894de146dace15a0)

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md (hunk 7 above)
Comment (ayeshLK) [reply downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md]: Fixed with [21ba8dd8c3a061eb1718a6a5894de146dace15a0](https://github.com/ballerina-platform/ballerina-dev-website/pull/9221/commits/21ba8dd8c3a061eb1718a6a5894de146dace15a0)

**Review by Nuvindu — APPROVED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 8]
```diff
@@ -197,58 +264,4 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
...
-
- A bug that caused an invalid static type to be set for optional XML attribute access on `xml:Element` has been fixed for compliance with the specification. The static type now includes `error`.
-
-```ballerina
-public function main() {
-    xml:Element xe = xml `<x attr="e"/>`;
-    string? attr = xe?.attr; // Compile-time error now.
-}
-```
-
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:210]: Incorrect change.

**Comment by ayeshLK** [comment]
Closing this PR as the changes were added in a separate PR [1]

[1] - https://github.com/ballerina-platform/ballerina-dev-website/pull/9232
